src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/ConfirmedAppointmentDetailsPage.jsx

```diff
@@ -45,11 +43,11 @@ function formatAppointmentDate(date) {
 }
 
 function formatHeader(appointment) {
-  if (isVideoGFE(appointment)) {
+  if (appointment.videoData?.kind === VIDEO_TYPES.gfe) {
     return 'VA Video Connect using VA device';
-  } else if (isVideoVAFacility(appointment)) {
+  } else if (appointment.videoData?.kind === VIDEO_TYPES.clinic) {
     return 'VA Video Connect at VA location';
-  } else if (isAtlasLocation(appointment)) {
+  } else if (appointment.videoData?.isAtlas) {
     return 'VA Video Connect at an ATLAS location';
   } else if (isVideoHome(appointment)) {
```

Comment:
This is an example of a helper I left alone, because it checks a combination of video properties